index.html

```diff
@@ -36,7 +37,129 @@ <h1>I’m Alexandra.<br>Glad to see you!</h1>
                             <li><a href="#"><img src="img/github.jpg" alt="github icon"></a></li>
                             <li><a href="#"><img src="img/angellist.jpg" alt="angellist icon"></a></li>
                         </ul>
-        </section>  
+        </div>
+    </section>  
+            <!-- the first card............... -->
+            <section id="work-section">
+                <div class="cards">
+                    <img src="img2/Snapshoot Portfolio.png" alt="portfolio img">
+                    <div class="align-text">
+                    <h2 class="h2-size">Tonic</h2>
+                    <ul class="first-list">
+                        <li class="list1">CANOPY</li>
+                        <li class="list2">Back End Dev</li>
+                        <li class="list2">2015</li>
+                    </ul>
+                    <P class="work-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                    <ul class="blue-list">
+                        <li>HTML</li>
+                        <li>Css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <button class="work-btn btn">see project</button>
+                    </div>
+                </div>
+            <!-- the second card ...............-->
+                        <div class="cards">
+                            <img src="img2/Snapshoot Portfolio2.png" alt="portfolio img">
+                        <div class="align-text" id="align-id">
+                            <h2 class="h2-size" >Multi-Post<br> Stories</h2>
+                            <ul class="first-list">
+                                <li class="list1">CANOPY</li>
+                                <li class="list2">Back End Dev</li>
+                                <li class="list2">2015</li>
+                                </ul>
+                                <P class="work-p">A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                                <ul class="blue-list">
+                                    <li>HTML</li>
+                                    <li>Css</li>
+                                    <li>javascript</li>
+                                </ul>
```

Comment:
[OPTIONAL]
- [ ] from the figma template you can see that each technology has a background colour please make sure to check the figma template and implement this background color to your project. :+1: 
- figma design
![f-t-p-p](https://user-images.githubusercontent.com/89055502/235338099-496e5b21-418b-447d-9559-27c36befae3b.png)
- your design
![s-t-p-p](https://user-images.githubusercontent.com/89055502/235338065-ee2d4b46-a186-4e6a-a872-5903b225bbc2.png)